README.rst

```diff
@@ -46,6 +46,7 @@ Configure pillar data under salt:ssh_roster to feed the template.
 
 Install salt api
 Requisite: Configure salt-master with rest_cherrypy or rest_tornado.
+Requires: pip.extensions as it installs the latest version from pip.
```

Comment:
I'm not a huge fan of having an external dependency. What does this buy us? Why can't we just do a pip.installed instead of pulling in an outside state from another formula?